chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies

```diff
@@ -1,5 +1 @@
 node-gyp-headers
```

Comment:
So is this entirely extraneous as it is currently and can just be removed without anything else needing to be done or is there other things that need doing to be able to remove this?